src/test/java/com/znsio/api/jsonPlaceHolder.feature

```diff
@@ -0,0 +1,76 @@
+      | Search by invalid title:<title>                  | ''     | ''     | @testAlphnumer!cT!tl3                                                        | ''                                                                                                                                                                  | 404            | negativeCaseResponseSchema |
+      | Search by null title:<title>                     | ''     | ''     | 'null'                                                                       | ''                                                                                                                                                                  | 404            | negativeCaseResponseSchema |
+      | search by partial title match: <title>           | ''     | ''     | 'dolorem dolore est'                                                         | ''                                                                                                                                                                  | 404            | negativeCaseResponseSchema |
+      | Search by null body:<body>                       | ''     | ''     | ''                                                                           | 'null'                                                                                                                                                              | 404            | negativeCaseResponseSchema |
+      | Search by invalid body:<body>                    | ''     | ''     | ''                                                                           | @testAlphnumer!cb0d¥                                                                                                                                                | 404            | negativeCaseResponseSchema |
+      | search by partial body match: <body>             | ''     | ''     | ''                                                                           | 'dignissimos aperiam dolorem'                                                                                                                                       | 404            | negativeCaseResponseSchema |
+
+  Scenario Outline: To verify functionality of GET /comments endpoint
+    * def positiveCaseResponseSchema = {postId:#number , id:#number, name:'#string', email:'#string',body:'#string'}
+    * def negativeCaseResponseSchema = []
+    * def query = {postId:<postId> , id:<id>, name:<name>, email:<email>,body:<body>}
+    * callonce read('classpath:com/znsio/common/JsonPlaceholder.feature@buildQuery')
+    * def builtQuery = buildQuery(query)
+    Given def getCallResponse = karate.call('classpath:com/znsio/templates/jsonPlaceHolderTemplates.feature@t_getCommentsByParams',{ params: builtQuery, expectedStatus: <expectedStatus>}).response
+    Then karate.log("comments <TestCase>",getCallResponse)
+    Then match each getCallResponse contains query
```

Comment:
Why you have used match each here?

Reply:
Using match each because for  query "postId:1",we get array with multiple elements having "postId:1"